lib/salus.rb

```diff
@@ -7,10 +7,10 @@
     config.endpoint = ENV.fetch('BUGSNAG_ENDPOINT', 'notify.bugsnag.com')
     config.api_key = ENV['BUGSNAG_API_KEY']
   end
-end
 
-# Hook at_exit to send off the fatal exception if it occurred
-at_exit { Bugsnag.notify($ERROR_INFO) if $ERROR_INFO }
+  # Hook at_exit to send off the fatal exception if it occurred
+  at_exit { Bugsnag.notify($ERROR_INFO) if $ERROR_INFO }
```

Comment:
any reason we use `at_exit` for this 🤔?  I didn't notice this before.

Reply:
$ERROR_INFO represents unhandled exceptions.  This only place to ensure all such exceptions are reported is the program exit point.
This call has been in salus since the first commit of salus.rb.